src/locales/en_US.js

```diff
@@ -146,8 +146,10 @@ const strings = {
   create_wallet_import_title: 'Import Wallet',
   create_wallet_import_instructions: 'Enter your private seed to verify and restore the associated wallet:',
   create_wallet_import_input_prompt: 'Private Seed',
-  create_wallet_import_input_key_or_seed_instructions: 'Enter the private key or mnemonic seed to import the associated wallet',
-  create_wallet_import_input_key_or_seed_prompt: 'Private Key or Mnemonic Seed',
+  create_wallet_import_key_instructions: 'Enter your private key to verify and restore the associated wallet:',
```

Comment:
Why is there a ':' at the end of the line?
It wasn't in the task on Asana. Remove it.